flixel/system/FlxSound.hx

```diff
@@ -442,11 +447,41 @@ class FlxSound extends FlxBasic
 	 */
 	public inline function stop():FlxSound
 	{
-		cleanup(autoDestroy, true, true);
+		cleanup(autoDestroy, true);
 		return this;
 	}
 	
 	/**
+	 * Sets the sound point at which to restart playing after a loop
+	 * @param	newloopPoint The new position in the sound, in milliseconds
+	 * @return	The updated value in milliseconds
+	 */
+	public function set_loopPoint(newloopPoint:Float):Float
+	{
+		if (looped && playing && loopPoint == 0.0)
+		{
+			loopPoint = newloopPoint;
+			var pos:Float = _channel.position;
+			cleanup(false, false);
+			startSound(pos);
+			return loopPoint;
+		}
+		loopPoint = newloopPoint;
```

Comment:
I think: you need to spot the case in which the sound is playing, is looping, and has no loopPoint (starts from the beginning). In that case, it meant it was played (startSound()) with MAX_INT as number of loops, which means that it will never stop. In this case, you have to update the channel to set the num of loops to be 0, so that after each completed play "stopped" is called and you can call then "play" with the time parameter. It' similar to what "set_looped" does.